### PR TITLE
Additional text new text field size with migration valerii

### PR DESCRIPTION
**Pull request by ValDekh**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.DAL/Persistence/Migrations/20230511081334_NewSizeOfTextFields.Designer.cs
Comment (ormykhalyshyn) [inline Streetcode/Streetcode.DAL/Persistence/Migrations/20230511081334_NewSizeOfTextFields.Designer.cs:1]: you need to do only one migration not 4 or 5

**Review by ormykhalyshyn — CHANGES_REQUESTED** [review]
Only one migration, not 6, also change the connection string when making a pull request.

File: Streetcode/Streetcode.DAL/Persistence/Migrations/20230511081334_NewSizeOfTextFields.Designer.cs
Comment (ValDekh) [reply Streetcode/Streetcode.DAL/Persistence/Migrations/20230511081334_NewSizeOfTextFields.Designer.cs:1]: done

File: Streetcode/Streetcode.DAL/Persistence/Migrations/StreetcodeDbContextModelSnapshot.cs
Comment (ormykhalyshyn) [inline Streetcode/Streetcode.DAL/Persistence/Migrations/StreetcodeDbContextModelSnapshot.cs:1]: You must rename this migration. Firstly, you must remove this migration and name it Initial.

**Review by ormykhalyshyn — CHANGES_REQUESTED** [review]

File: Streetcode/Streetcode.DAL/Persistence/Migrations/StreetcodeDbContextModelSnapshot.cs
Comment (ValDekh) [reply Streetcode/Streetcode.DAL/Persistence/Migrations/StreetcodeDbContextModelSnapshot.cs:1]: Done

**Review by Dobriyr — CHANGES_REQUESTED** [review]
Looks nice but need to add some changes

File: Streetcode/StreetCode.Client
[the remark this answers is not in the document]
Comment (ValDekh) [reply Streetcode/StreetCode.Client:1]: Done

File: Streetcode/Streetcode.WebApi/Controllers/Streetcode/StreetcodeController.cs [hunk 1]
```diff
@@ -93,7 +93,14 @@ public async Task<IActionResult> GetByIndex([FromRoute] int index)
...
     public async Task<IActionResult> Create([FromBody] StreetcodeCreateDTO streetcode)
     {
-        return HandleResult(await Mediator.Send(new CreateStreetcodeCommand(streetcode)));
+        try
+        {
+            return HandleResult(await Mediator.Send(new CreateStreetcodeCommand(streetcode)));
+        }
+        catch (Exception)
+        {
+            return BadRequest();
```
[the remark this answers is not in the document]
Comment (ValDekh) [reply Streetcode/Streetcode.WebApi/Controllers/Streetcode/StreetcodeController.cs]: Done

File: Streetcode/Streetcode.WebApi/appsettings.Local.json
[the remark this answers is not in the document]
Comment (ValDekh) [reply Streetcode/Streetcode.WebApi/appsettings.Local.json:1]: Done

File: Streetcode/Streetcode.WebApi/appsettings.json
[the remark this answers is not in the document]
Comment (ValDekh) [reply Streetcode/Streetcode.WebApi/appsettings.json:1]: Done

**Pull request closed without merging**